composer.json

```
@@ -40,7 +40,7 @@
         "test-cov": "\"vendor/bin/phpunit\" --coverage-text",
         "test-group": "\"vendor/bin/phpunit\" --coverage-text --group",
         "test-ci": "\"vendor/bin/phpunit\" --coverage-clover=coverage.xml",
-        "pre-commit": [ "@phpcs", "@test" ]
+        "pre-commit": [ "@phpcs", "@test", "@composer validate --strict" ]
```

```suggestion
        "pre-commit": [ "@composer validate --strict", "@phpcs", "@test" ]
```

The validation should be caught before the tests run.